be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/SocialMediaValidator.scala

```diff
@@ -33,4 +33,6 @@ case object SocialMediaValidator extends MessageDataContentValidator {
             case _ => Right(validationErrorNoMessage(rpcMessage.id))
         }
     }
-}
\ No newline at end of file
+
+    //no need for a addBroadcastChirpValidator, since the server will never receive one in theory
```

Comment:
I am not very sure about that. I do not know exactly how the consensus project works. Maybe servers validate message they receive before trying to reach consensus with other servers. Could @pierluca tell us more about this (i.e. should we validate thoroughly such messages, or not)

In the general case, I would still have a validator for every type of messages (even if it does nothing or returns an error).

Reply:
I'll add an "useless" validator for now